x-pack/plugins/cases/common/api/cases/comment/index.test.ts

```diff
@@ -777,6 +778,29 @@ describe('Comments', () => {
         right: defaultRequest,
       });
     });
+
+    it('error when too many comments are passed', () => {
+      const query = BulkCreateCommentRequestRt.decode(
+        Array(MAX_ADD_COMMENTS + 1).fill({
+          comment: 'Solve this fast!',
+          type: CommentType.user,
+          owner: 'cases',
+          foo: 'bar',
+        })
+      );
+
+      expect(query).toEqual({
+        _tag: 'Left',
+        left: expect.any(Array),
```

Comment:
Maybe you can use `PathReporter` to check error message like this:

```
 it('throws error when comment is too long', () => {
          const longComment = 'x'.repeat(MAX_COMMENT_LENGTH + 1);
          expect(
            PathReporter.report(
              CommentRequestRt.decode({ ...defaultRequest, comment: longComment })
            )
          ).toContain('The length of the comment is too long. The maximum length is 30000.');
        });
```

Reply:
This is much better, i didn't know about `PathReporter` 😅